content/10.diagnostics.md

```diff
@@ -28,6 +30,13 @@ In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2
 Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
 However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
 
+#### Pooled PCR Testing Approaches
+
+Due to limited supplies and the need for more tests, a number of labs have found ways to enable pooled or similar tests.
+The first such result was a group at the Technion, which found they could pool up to 32 samples [@doi:10.1101/2020.03.26.20039438], followed by larger pooling in work utlizing slightly different methods at the University of Minnesota [@doi:10.1101/2020.04.02.022186].
```

Comment:
```suggestion
The first such result came from a group at the Technion, which found they could pool up to 32 samples [@doi:10.1101/2020.03.26.20039438], which was then followed by larger-scale pooling in work that utilized slightly different methods at the University of Minnesota [@doi:10.1101/2020.04.02.022186].
```
Are these both RT-PCR with minor methodological changes, or is something more fundamental changing? Just in case it's helpful, FYI we have a section about dPCR at #206 that should be merged into the manuscript quite soon! Also, I'm wondering if larger pooling is "larger-scale pooling", or maybe "higher-throughput pooling"?